internal/pkg/service/cli/dialog/allowed_branches_test.go

```diff
@@ -25,17 +26,21 @@ const (
 // TestAllowedBranchesByFlag use flag value if present.
 func TestAskAllowedBranchesByFlag(t *testing.T) {
 	t.Parallel()
-	dialog, o, console := createDialogs(t, true)
+	dialog, _, console := createDialogs(t, true)
 	d := dependencies.NewMocked(t)
 	registerMockedBranchesResponse(
 		d.MockedHTTPTransport(),
 		[]*keboola.Branch{{BranchKey: keboola.BranchKey{ID: 123}, Name: "Main", IsDefault: true}},
 	)
-	o.SetDefault(`branches`, `*`)
-	o.Set(`branches`, `foo, bar`)
+	// o.SetDefault(`branches`, `*`)
+	// o.Set(`branches`, `foo, bar`)
+
+	f := syncInit.Flags{
+		Branches: configmap.NewValueWithOrigin("foo, bar", configmap.SetByFlag),
+	}
 
 	// No interaction expected
-	allowedBranches, err := dialog.AskAllowedBranches(context.Background(), d)
+	allowedBranches, err := syncInit.AskAllowedBranches(context.Background(), d, dialog, f)
 	assert.NoError(t, err)
 	assert.Equal(t, model.AllowedBranches{"foo", "bar"}, allowedBranches)
```

Comment:
You moved the dialogs to individual commands, but you left the tests in the "dialog" package, which is not good.

The tests should be with the code they are testing, can you please move it?

**This search should be finally empty,** no command package should be imported anywhere in the dialog tests.

![image](https://github.com/keboola/keboola-as-code/assets/19371734/c9e21e08-81e5-4330-a194-247521e81d08)